src/components/select/list-action-button/list-action-button.component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import StyledListActionButtonWrapper from './list-action-button.style';
+import Button from '../../button';
+
+const ListActionButton = React.forwardRef(({ listActionButton, onListAction }, ref) => {
+  const listActionButtonDefaultText = I18n.t(
+    'select.action_button_text',
+    {
+      defaultValue: 'Add New Item'
+    }
+  );
+
+  function renderListActionButton() {
+    if (!listActionButton || listActionButton === true) {
+      return (
+        <Button
+          forwardRef={ ref }
+          onClick={ onListAction }
+          iconType='add'
```

Comment:
Will this always be the `add` icon?

Reply:
yes, as a default. You could provide your own button with your own icon.

Reply:
Apologies I didn't read that properly this time, it's clearly a default!